src/EnergyPlus/SimulationManager.cc

```diff
@@ -1547,6 +1555,14 @@ namespace SimulationManager {
 		}
 		mtr_stream = nullptr;
 
+		if ( ReportExtShadingSunlitFrac ) {
+			gio::write( OutputFileShadingFrac, EndOfDataFormat );
+			gio::close( OutputFileShadingFrac );
+		}
+		else {
+			{ IOFlags flags; flags.DISPOSE( "DELETE" ); gio::close( OutputFileShadingFrac, flags ); }
```

Comment:
What logic is feeding into this deletion??  This makes me nervous.  It looks like you are saying that if we aren't reporting new values, delete the old file.  So then if a user is tweaking things around with a long running model, and they turn off the shading stuff once while tweaking, it's going to delete their file.